### PR TITLE
Update de_de.json (1.16.5)

### PR DESCRIPTION
**Pull request by DaLumma**
Updated the german translation for 1.16.5

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/de_de.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
   "block.farmersdelight.basket": "Korb",
   "block.farmersdelight.cutting_board": "Schneidbrett",
   "block.farmersdelight.skillet": "Bratpfanne",
-  "block.farmersdelight.oak_pantry": "Eichenholzküchenschrank",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/de_de.json]: It seems you used the new 1.17+ keys in this lang. 1.16 still uses the old keys, such as `pantry` and `rice_crop`.
All diff lines that change keys are affected, so it should be easy to rollback.

**Review by vectorwing — CHANGES_REQUESTED** [review]
Thanks for the update! I commented on an issue with the lang keys; we can merge once resolved.

File: src/main/resources/assets/farmersdelight/lang/de_de.json (hunk 1 above)
Comment (DaLumma) [reply src/main/resources/assets/farmersdelight/lang/de_de.json]: they should now be back to the old ones didn't notice that they stayed untouched in 1.16

**Review by vectorwing — APPROVED** [review]
Nicely done! 👍

**Pull request merged**